Amplifier_Current.lib

```diff
@@ -688,7 +709,7 @@ $FPLIST
 $ENDFPLIST
 DRAW
 S -300 300 400 -200 0 1 10 f
-X REF 1 500 0 100 L 50 50 1 1 I
+X GND 1 100 -300 100 U 50 50 1 1 W
```

Comment:
I don't think you meant to change this pin :)

Reply:
I don't think I did either.. Thanks